### PR TITLE
Make manifest steps clearer

### LINKED ISSUE #778: Step 50 in Test Manifest is listed incorrectly

### LINKED ISSUE #779: Update step 126 in Test Manifest to reflect maintainer workflow
It's currently:
> Clicking on a pull request title opens browser window to pull request on .com

But now clicking on the title displays a detailed view of the pull request

### LINKED ISSUE #780: Update 'Test 70's decription of the "New Project" dialog

### LINKED ISSUE #781: Update 'Test 77's description of "Publish to Github" pane

### LINKED ISSUE #782: 'Test 78's field list missing GitHub account type dropdown

### LINKED ISSUE #785: 'Test 103' should describe the expected error message better

### PR DESCRIPTION
**Pull request by paladique**
@jcansdale and @StanleyGoldman found some inconsistencies in the steps while working through the manifest for the 2.2.0.4 release. 

- [x] Fixes #778 
- [x] Fixes #779
- [x] Fixes #780 
- [x] Fixes #781 
- [x] Fixes #782 
- [x] Fixes #785

### CODE REVIEW

**Comment by StanleyGoldman** [comment]
_Humble pardons_
I updated your comment to make them into checkboxes.

File: documentation/manifest.md [hunk 1]
```diff
@@ -68,14 +69,15 @@
     - [ ] Team Explorer view changes to Home page
     - [ ] GitHub header and repo information is shown
 - [ ] **Publishing a local repo**
-  - [ ] File - New - Project - Console Application (or any type of project, doesn't matter much)
-  - [ ] Select "Add to source control" from the dialog and click Ok
-  - [ ] Select "Git" from the Choose Source Control dialog
+- [ ] **File - New - Project - Console Application (or any type of project, doesn't matter much)**
+- [ ] **Set "Create new Git Repository" checkbox to checked**
+- [ ] **Click OK button**
```
Comment (StanleyGoldman) [inline documentation/manifest.md]: Should this chunk be indented?

Comment (paladique) [inline documentation/manifest.md]: They should! They're all just setup steps for the next batch of testing

Comment (paladique) [inline documentation/manifest.md]: Ah, I see. They were previously indented and then they no longer are. I tried to keep the setup steps outdented the furthest, but if the steps look clearer with an indent, no objections here to do that!

Comment (paladique) [inline documentation/manifest.md]: Updated! 👍

**Comment by Haacked** [comment]
@paladique, this is labeled as "ready for review" but the issue has two task list checkboxes that are not checked. Unsure if those are still being worked on, are completed and just unchecked by accident, or are unnecessary. Could you clarify? Thanks!

**Comment by paladique** [comment]
@haacked @stanleygoldman added the checkboxes and also made a suggestion that I've pushed, so I think these changes might check them all off now.

**Comment by StanleyGoldman** [comment]
All good here. 👍

**Pull request merged**